### ISSUE TITLE
Taiga update detected as a trojan?

### ISSUE THREAD

**Issue by hopelessperson** [erengy/taiga#159]
![2015-08-10_14-12-05](https://cloud.githubusercontent.com/assets/13737849/9184427/1966249e-3f6a-11e5-99d8-473ba52c47e6.png)

**Comment by TheAlaine** [comment]
Same here.

**Comment by betatan** [comment]
Same here, using Nod32 Antivirus 8

**Comment by Injabie3** [comment]
8/55 detection on VirusTotal: https://www.virustotal.com/en/file/6ebd9254be96429b70d998830eabbb19cfe3d87834604a5ee968ad7b0e649cbb/analysis/1439256631/

**Comment by erengy** [comment]
It was 2/56 when I first submitted it, went ahead to 8/55 at night, and now it's [7/56](https://www.virustotal.com/en/file/6ebd9254be96429b70d998830eabbb19cfe3d87834604a5ee968ad7b0e649cbb/analysis/1439276071/)... I don't understand why they're classifying TaigaSetup.exe as a trojan, but not Taiga.exe itself.

I've reported the file as a false positive to [Avast](http://www.avast.com/contact-form.php?subject=VIRUS-FILE), [Avira](https://analysis.avira.com/en/submit), [Baidu](http://antivirus.baidu.com/en/submit-file.php), [ESET](http://support.eset.com/kb141/), [Kaspersky](http://newvirus.kaspersky.com) and [McAfee](https://community.mcafee.com/thread/2016). So far, only Kaspersky replied:

> Hello,

> Sorry, it was a false detection. It will be fixed in the next update.
> Thank you for your help.

> Sincerely yours,
> Chris Zachor,
> Junior Malware analyst.

Still waiting to hear back from the others (it can take up to 48 hours, they say). I'm tired of doing this on every new release, though (see: #19).

For those who're rightfully skeptical about the warnings, you can try running TaigaSetup.exe in [Sandboxie](http://www.sandboxie.com) and see for yourself: It just copies the files to the install location, and creates shortcuts on desktop and the Start menu. You can also examine the NSIS script of the setup [here](https://github.com/erengy/taiga/blob/4021e70c41a29477fad229d840ce8d28beb8b8ff/setup/Taiga.nsi).

**Comment by TheAlaine** [comment]
First thank you very much for this programm, it is a blast and i dont want to manage my anime without it anymore. Glad to here it was a false alarm. And thanks for the quick reply :)

**Comment by Aurielle** [comment]
Having the same problem, thanks for the false positive reports. Keep up the good work :+1:

**Comment by kie8** [comment]
Having the same problem but with bitdefender. It deleted taiga.exe and I can't even re-install it.




![untitled](https://cloud.githubusercontent.com/assets/13747768/9197101/18ae380e-403a-11e5-91c6-28965d7021cb.png)

**Comment by TheAlaine** [comment]
Btw after the new signature update on Eset nod 32 you cant even download it anymore, it just blocks the download site. Had to disable the web protection to get it. But then i could install it without problems from eset.

**Comment by erengy** [comment]
@kie8
That's weird because [VirusTotal says](https://www.virustotal.com/en/file/9fffcb2ceeb205524decc1408954e7e1102656aa03fe84573e0ae4bb43ed0e6e/analysis/) Taiga.exe is clean according to BitDefender. In any case, I sent them an email now.

@TheAlaine
NOD32 no longer classifies TaigaSetup.exe as malicious, but now it thinks the download link is... I just sent another mail to them, requesting the domain to be whitelisted.

**Comment by betatan** [comment]
I added Taiga's folder to exceptions, it's faster. Dunno why nod detects it as this: http://www.virusradar.com/en/Win32_Spy.Zbot.AAO/description

**Comment by Injabie3** [comment]
Thanks @erengy, keep us posted :+1:

**Comment by 09eragera09** [comment]
@erengy I use windows defender, and so far I haven't had any problems. Well windows defender is shit, but atleast I can use taiga in peace.

**Comment by erengy** [comment]
Symantec products were detecting Taiga as ["WS.Reputation.1"](https://www.symantec.com/security_response/writeup.jsp?docid=2010-051308-1854-99), so I submitted an [erroneous detection](https://submit.symantec.com/false_positive/) form and received the following response within an hour:

> Upon further analysis and investigation we have verified your submission and, as such, the detection(s) for the following file(s) will be removed from our products:

>        EFD2EB2C5F2DB03772E4D64914F28FE9 - Taiga.exe


> The updated detection(s) will be distributed in the next set of virus definitions, available via LiveUpdate or from our website at http://securityresponse.symantec.com/avcenter/defs.download.html

> Please note that whitelisting can take up to 24 hours to take effect.

**Comment by Aurielle** [comment]
ESET has already whitelisted the domain and no longer detecting the setup as a virus. Great work :+1:

**Comment by viddypiddy** [comment]
Windows Defender is detecting it for me now, on Windows 10! Darnit :')

**Comment by erengy** [comment]
Is your virus and spyware definitions up to date? Just tried scanning the files with Windows Defender (definition version 1.203.2099.0) and it didn't find any issues.

**Comment by ConnorKrammer** [comment]
I'm on Windows 7 using Windows Defender. My definitions are up to date, and Taiga hasn't been flagged as a virus on any of the instances I've updated it.

**Comment by viddypiddy** [comment]
My definitions are 1.203.2137.0, even after a manual update, and its still being flagged as malware.

**Comment by erengy** [comment]
Here's our current situation: **Avast, Avira, Baidu, ESET, Kaspersky and Symantec no longer detect the latest version of Taiga as malicious.** McAfee, Qihoo and Rising still do.

McAfee has changed the classification from "Artemis!A6E427EEE0BF" to "RDN/PWSZbot-FHN", but still insists that the setup is malware. I've already mailed them twice, don't know another try would help.

Qihoo is not exactly a reputable company [according to what I've read](http://www.pcworld.com/article/2919554/tencent-qihoo-antimalware-firms-are-accused-of-cheating-stripped-of-rankings-in-antivirus-tests.html), but I sent them a report anyway. I *think* they accepted the false positive:

> Dear Sir or Madam,

> The file that you’ve submitted has been identified as malicious file(Time: 2015-08-13 16:20:03; Software: TaigaSetup; ID:2058250).

> We sincerely appreciate your help of improving our products and services.

> Result: Proper actions have been taken. If the false positive happens again, please add it into local Trust List and contact us again with support@360safe.com .

> Thanks for your support.

Rising, another Chinese firm, doesn't even have a proper English website. They do have a submission page, though. I've just reported the file, we'll see if it works out.
